src/test/java/org/apache/maven/plugins/jlink/JLinkMojoTest.java

```diff
@@ -138,7 +139,12 @@ void testGetModulePathElements() throws Exception {
         outputDirectoryField.setAccessible(true);
         outputDirectoryField.set(mojo, outputDirectory);
 
-        Map<String, File> modulePathElements = mojo.getModulePathElements();
+        // Use reflection to access the private method
```

Comment:
Not really an improvement. Now the test is harder to understand. Is it possible to test it with an integration test (aka invoker test)?

@bmarwell Does exist a Maven guideline how test should look?